pkg/cmd/server/start/admission.go

```diff
@@ -41,6 +41,7 @@ var (
 
 func init() {
 	defaultOffPlugins.Insert("AlwaysPullImages")
+	defaultOffPlugins.Insert("JenkinsBootstrapper")
```

Comment:
> Allow jenkins auto-creation to be disabled from a single spot.

item 2 is still possible using the admission on/off mechanism


> Default the entire config so an admin can see what is used

This is a general problem with admission plugins, why should jenkins be special in this case?  The whole stanza has been reduced to an admission plugin (I think).